dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -69,9 +71,9 @@ public FileResponse readFile(String fileName, String reference) {
             Repository repo = service.getRepository(gitUsername, gitRepository);
             List<RepositoryContents> contents;
             try {
-                contents = cService.getContents(repo, fileName, reference);
+                contents = checkNotNull(cService.getContents(repo, fileName, reference));
```

Comment:
Is that correct? I think this checks that `cService.getContents(repo, fileName, reference)` is not null rather than that `fileName` was not null, which was the issue